donkeycar/parts/oled.py

```diff
@@ -98,11 +98,14 @@ def run(self, recording, num_records, user_mode):
             self.display.cls()
             self.update_slots()
             self.display_text()
-            self.display_user_mode_icon(user_mode)
+            self.display_user_mode_icon()
             self.display.display()
             self.update_requested = False
         self.display.process_queues()
 
+    def update(self):
```

Comment:
I think you can remove this method ?

Reply:
From my understanding of threaded parts, I cannot remove the method. Since the OLED part runs threaded with the latest update, the `update()` method is called from the donkey frame work. Thus from my understanding the part does not provide any output data, the body is empty. But maybe I lack understanding here. Maybe the parts [documentation](https://github.com/autorope/donkeycar/blob/dev/docs/parts/about.md) needs some updates at this point with more clarification on threaded parts